Makefile

```diff
@@ -90,7 +90,7 @@ oc-rm-dev:
 
 oc-forward-ports:
 	-make oc-stop-forwarding-ports 2>/dev/null
-	oc port-forward $$(oc get pods -o jsonpath='{.items[*].metadata.name}' -l name=masu-pgsql) 15432:5432 >/dev/null 2>&1 &
+	oc port-forward $$(oc get pods -o jsonpath='{.items[*].metadata.name}' -l name=koku-pgsql) 15432:5432 >/dev/null 2>&1 &
```

Comment:
Am I correct in interpreting this as masu won't be a standalone app, that you're expecting the dev environment to depend on the koku-pgsql DB?